torch/_inductor/decomposition.py

```diff
@@ -1105,3 +1105,11 @@ def rrelu_with_noise_functional(
     else:
         negative_slope = (lower + upper) / 2
         return aten.leaky_relu(self, negative_slope), torch.Tensor()
+
+
+@register_decomposition(aten._rms_norm_fused)
```

Comment:
We don't want this to happen. We want inductor to use the fused implementation, not to decompose it.